akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerMTSpec.scala

```diff
@@ -14,16 +14,16 @@ import akka.testkit._
 class CircuitBreakerMTSpec extends AkkaSpec {
   implicit val ec: ExecutionContextExecutor = system.dispatcher
   "A circuit breaker being called by many threads" must {
-    val callTimeout = 2.second.dilated
+    val callTimeout = 2.seconds.dilated
     val resetTimeout = 3.seconds.dilated
     val maxFailures = 5
-    val breaker = new CircuitBreaker(system.scheduler, maxFailures, callTimeout, resetTimeout)
+    def newBreaker = new CircuitBreaker(system.scheduler, maxFailures, callTimeout, resetTimeout)
```

Comment:
I'm not sure this is a fix though, but thought it was wise to have distinct instances per tests because the resetTimout is multiplied by the backoff factor.